### PR TITLE
Add new Ninja logo and improve typography & styling of documentation

### PR DESCRIPTION
**Pull request by gitblit**
I've created an SVG from the shape of the console ASCII logo and remixed it with a
Ninja star.  I also created a square logo for social media accounts, favicons, and
other purposes.

I believe the css styling tweaks, font changes, and introduction of FontAwesome
produce a better-formatted website.

Obviously all of this is subjective.

### CODE REVIEW

**Comment by gitblit** [comment]
homepage

**Comment by pedro-stanaka** [comment]
Wow, that looks nice...

File: ninja-core/src/site/site.xml [hunk 1]
```diff
@@ -13,16 +13,18 @@
...
+
+            <meta name="description"
+                  content="Ninja is a full stack web framework for Java.
                     Rock solid, fast and super productive."/>
             <link rel="icon" type="image/png" href="favicon.png" />
 
-            <link href='http://fonts.googleapis.com/css?family=Droid+Sans:400,700|Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'/>
-    
+            <link href='http://fonts.googleapis.com/css?family=Titillium+Web' rel='stylesheet' type='text/css'/>
+            <link href='http://fonts.googleapis.com/css?family=Incolsolata' rel='stylesheet' type='text/css'/>
```
Comment (raphaelbauer) [inline ninja-core/src/site/site.xml]: That's a tiny typo imho...  should link to http://fonts.googleapis.com/css?family=Inconsolata

**Comment by gitblit** [comment]
Typo fixed.

**Comment by raphaelbauer** [comment]
Hey Gitblit,

so I was thinking about your new logo and design. And I really like the new styling, fonts and the logos you created :)

There are just two things to consider:
1) People may use your logo for a lot of things. Even commercial things. It's Apache 2 licensed after all.
2) It may be the case that someone proposes a new design / logo. And it may be the case that it will replace your logo - even though you think it looks awful and ugly - and - beware - without even asking. Whoever has commit rights at that point may just merge the new design.

That's nothing bad :) It just means what you said at the very beginning of your PR:  "Obviously all of this is subjective.". And I think we should not fight about that. Let's fight about reloading strategies ;)

After all you are replacing my initial design (with something nicer I have to admit ;) )...

ra

**Comment by gitblit** [comment]
This is not a problem for me.  People already take stuff I authored, remix it ,and embed it in commercial products.  That's a part of the ecosystem.  I do the same thing.  :)

Instead of replacing your design, I'd rather think of it as embracing and enhancing your original design.  IIRC the ascii boot logo predates the (more recent) website redesign.

```
 _______  .___ _______        ____.  _____   
 \      \ |   |\      \      |    | /  _  \  
 /   |   \|   |/   |   \     |    |/  /_\  \ 
/    |    \   /    |    \/\__|    /    |    \
\____|__  /___\____|__  /\________\____|__  /
     web\/framework   \/                  \/ 
```

**Comment by raphaelbauer** [comment]
great!

**Comment by Zofren** [comment]
If I may, I prefer the previous logo. This new one is nice looking (and the shuriken idea is great) but not "iconifiable" and to be honest, it doesn't look professional (to me !) on the main web site. There is something to do with the banner typo but the logo was already nice.

**Comment by gitblit** [comment]
@Zofren I can see why you feel that way.  The graphic I proposed in this PR has a young/urban/graffiti feel to it which you might say looks unprofessional.  As I stated in the proposal, artwork is subjective.

In regards to  scalablity, I did provide a square SVG variant which should be "iconifiable" comfortably down to 32x32.

For reference, this is the 512x512 social media icon that was included in this PR.
ninja-square

**Comment by raphaelbauer** [comment]
@Zofren I understand your concerns.

The good news is that that will not be the last change to the Ninja webpage :) We should make a vote next time - just to make sure all voices get heard :)

**Pull request merged**